pkg/compose/stop.go

```diff
@@ -21,29 +21,31 @@ import (
 
 	"github.com/docker/compose/v2/pkg/api"
 	"github.com/docker/compose/v2/pkg/progress"
-
-	"github.com/compose-spec/compose-go/types"
+	//"github.com/compose-spec/compose-go/types"
```

Comment:
nit : remove if not necessary

Reply:
You're very welcome.

> Can you add the same behaviour when using the `--project-name` ?

Added this error check to `start`, `stop` and `restart`

> nit : remove if not necessary

Done.


Also added `service.Scale` to `projectFromName` to maintain compatibility with https://github.com/docker/compose/commit/8a9498c5710c8282dba42241b3217d8a0a366ff9